build/cmake/AddMCRL2Binary.cmake

```diff
@@ -175,38 +181,55 @@ function(_add_mcrl2_binary TARGET_NAME TARGET_TYPE)
      (IS_GUI_BINARY AND NOT MCRL2_ENABLE_GUI_TOOLS))
     return()
   endif()
+ 
+  get_filename_component(SOURCEDIR_ABS ${ARG_SOURCEDIR} ABSOLUTE)
+  get_filename_component(INCLUDEDIR_ABS ${ARG_INCLUDEDIR} ABSOLUTE)
 
-  foreach(SRC ${ARG_SOURCES})
-    get_filename_component(SRC_ABS ${SRC} ABSOLUTE)    
-    if(NOT "${SRC_ABS}" STREQUAL "${SRC}")
-      get_filename_component(SRC_ABS ${ARG_SOURCEDIR}/${SRC} ABSOLUTE)
+  foreach(_SRC ${ARG_SOURCES})
```

Comment:
I don't think we are using this underscore prefix convention anywhere else, so probably best to just keep it consistent with what is already there.